scripts/dev_fd_quadrature.jl

```diff
@@ -0,0 +1,118 @@
+# Script to get a feeling for speed vs accuracy of quadrature schemes
```

Comment:
We can afford to be very generous with quadrature points, since `muladd` is cheap and fully vectorized.

Reply:
Currently, the actual cosmology calculation is very subdominant to the linear algebra going on in the ODE solver.

Reply:
Yeah makes total sense - not actually an interesting thing to check for performance. Was wondering aloud, but what I guess I meant was how much is the error actually changing since I don't have a good feel for it.